testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/BaseFailureRecoveryTest.java

```diff
@@ -325,6 +325,23 @@ public void testAnalyzeTable()
                 false);
     }
 
+    public void testMerge()
+    {
+        testTableModification(
+                Optional.of("CREATE TABLE <table> AS SELECT * FROM orders"),
+                """
+                        MERGE INTO <table> t
+                        USING (SELECT orderkey, 'X' clerk FROM <table>) s
+                        ON t.orderkey = s.orderkey
+                        WHEN MATCHED AND s.orderkey > 1000
+                            THEN UPDATE SET clerk = t.clerk || s.clerk
+                        WHEN MATCHED AND s.orderkey <= 1000
+                            THEN DELETE
+                        """,
+                Optional.of("DROP TABLE <table>"));
+    }
+
```

Comment:
Delete the extra blank line. It fails checkstyle.

Reply:
Sorry; fixed.